src/main/java/org/testcontainers/containers/CephContainer.java

```diff
@@ -39,6 +42,8 @@ public class CephContainer extends GenericContainer<CephContainer> {
 
     private static final String CEPH_END_START_REGEX_FORMAT = ".*Bucket 's3://%s/' created\n.*";
 
+    private static final Ulimit[] DEFAULT_ULIMITS = new Ulimit[]{new Ulimit("nofile", 65536L, 65536L)};
```

Comment:
can you explain this to me. What this ulimit stuff means in layman terms. I think i know, i just want to confirm we are on the same page.